transifex/pull.py

```diff
@@ -115,21 +120,22 @@ def update_translations(self):
         """
         subprocess.run(['make', 'pull_translations'], check=True)
 
-        # Messages may fail to compile (e.g., a translator may accidentally translate a
-        # variable in a Python format string). If this happens, we want to proceed with
-        # the PR process and notify the team that messages failed to compile.
-        try:
-            # The compilemessages command is a script that (as of Django 1.9) scans the project
-            # tree for .po files to compile and calls GNU gettext's msgfmt command on them. It
-            # doesn't require DJANGO_SETTINGS_MODULE to be defined when run from the project root,
-            # and also doesn't need django.setup() to be run. Because of this, we can get away with
-            # django-admin.py instead of manage.py. The latter defines a default value for
-            # DJANGO_SETTINGS_MODULE and causes django.setup() to run, which is undesirable here for reasons
-            # ranging from Python 2/3 incompatibility errors across projects to forcing the installation
-            # of packages which provide installed apps custom to each project.
-            subprocess.run(['django-admin.py', 'compilemessages'], check=True)
-        except subprocess.CalledProcessError:
-            self.is_compiled = False
+        if not skip_compile_messages:
```

Comment:
Instead of indenting everything, just return early:

```python
if skip_compile_messages:
  # TODO Log the fact that compilation is skipped
  return
```